src/commands/runDraftTool/runDraftCreate.ts

```diff
@@ -75,19 +73,17 @@ export default async function runDraftCreate(
          );
          const command = buildCreateCommand(destination, fileType, configPath);
 
-         const result = await runDraftCommand(command);
+         const [success, err] = await runDraftCommand(command);
```

Comment:
not sure if this is out of scope but can we have runDraftCommand return some sort of result object rather than just an array of two strings? feel like returning a clear success/stderr field will be easier to read and use